### PR TITLE
Exchange ID token for Apigee token on login.

### PR DESCRIPTION
**Pull request by sylmak**
Exchange ID token for Apigee token on login.

For review, compare against verify-email.

### CODE REVIEW

File: src/common.ts [hunk 1]
```diff
@@ -1,6 +1,83 @@
...
+  const response = await axios.post(
+    TOKEN_API_ENDPOINT_URI,
+    {
+      grant_type: 'urn:ietf:params:oauth:grant-type:jwt-bearer',
+      assertion: firebaseToken,
+    },
+    {
+      headers: {
+        Authorization:
+          'Basic cWNRZXNsbnY5eVFTS09kbkVEWDBCTkRVM2xjTzIwOWU6ZmFkVWZ5Q21OMDg0emJJcQ==',
```
Comment (skeshive) [inline src/common.ts]: Here is the new authorization code to use: cDF1QXM4T3NGdDI2emlUYWIzQmt2cFdNOUJIbDVyUmNjT25CZHhzVFIya3MwVEZ0OmNTNU1WSFFjTkI4VHJ0UXJLOVZpc09BNDFNUGVRQUtjWHc5ZmRPcTZIWFN4aDBUdU1Va0h5Tk9wZ0ZsYTBpYnI=

File: src/common.ts [hunk 2]
```diff
@@ -1,6 +1,83 @@
...
+  }
+  const response = await axios.post(
+    TOKEN_API_ENDPOINT_URI,
+    {
+      grant_type: 'urn:ietf:params:oauth:grant-type:jwt-bearer',
+      assertion: firebaseToken,
+    },
+    {
+      headers: {
+        Authorization:
```
Comment (skeshive) [inline src/common.ts]: Can you add a comment about why this is safe to expose?

TLDR: This authorization code is used for identifying this user authentication app. The Apigee /oauth/token endpoint will only return an API access token if a valid JWT issued by the authentication app is also present in the request.

**Review by skeshive — APPROVED** [review]
